uber/tests/models/test_watchlist.py

```diff
@@ -0,0 +1,56 @@
+from uber.tests import *
+
+
+@pytest.fixture()
+def session(request):
+    session = Session().session
+    request.addfinalizer(session.close)
+    setattr(session, 'watchlist_entry', session.watch_list(first_names='Banned, Alias, Nickname', last_name='Attendee'))
+    return session
+
+
+class TestIfBanned:
+    def test_no_last_name_match(self, session):
+        a = Attendee(first_name='Banned', last_name='Not')
+        session.add(a)
+        assert not a.banned
```

Comment:
I assume that you're saying ``session.add(a)`` in all of these functions because you're referencing ``a.banned`` which itself references ``a.watchlist_entry`` which is a foreign key relationship?

If so then you can probably get rid of this.  That should be ``None`` for new objects, so most of these functions are fine with ``.watchlist_entry`` being ``None``.  This test could be rewritten to simply read

```python
assert not Attendee(first_name='NotBanned', last_name='Attendee').banned
```

The only case where you actively need to follow the foreign key relationship is ``test_has_watchlist_entry`` below.  In that case you could just set ``watchlist_entry`` directly instead of setting the id, i.e.

```python
assert Attendee(watchlist_entry=session.watchlist_entry, first_name='Banned', last_name='Not').banned
```